### DISCUSSION TITLE
Use avif as openexr alternative for hdr movie maker

### DISCUSSION THREAD

**Discussion by octanejohn** [godotengine/godot-proposals#8940] [Engine Core]
webp, avi, png, jpeg doesnt support hdr 

use avif because openexr is bigger size, longer encode, limited support(browsers, editors, viewers, hardware acceleration, ect)

releated repos

https://github.com/AOMediaCodec/libavif

https://gitlab.com/AOMediaCodec/SVT-AV1

**Comment by Calinou** [comment]
- Related to https://github.com/godotengine/godot-proposals/issues/2461.

Unfortunately, AVIF's lossless compression is poor (it's slow *and* inefficient even compared to PNG), so I'd prefer having support for writing PNGs with high color depth instead. Remember that lossless output is preferable for advanced use cases, so I wouldn't opt for a format that is only good at lossy compression.
